src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,75 @@
+package core.basesyntax;
+
+import static org.junit.Assert.assertEquals;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.BeforeClass;
+import org.junit.Before;
+import org.junit.Test;
+
+public class RegistrationServiceImplTest {
+    private static RegistrationServiceImpl registrationServiceImpl;
+    private static User user;
+    
+    @BeforeClass
+    public static void beforeClass() {
+        registrationServiceImpl = new RegistrationServiceImpl();
+        user = new User();
+    }
+    
+    @Before
+    public void setUp() {
+        Storage.people.clear();
+        user.setId(12345L);
+        user.setLogin("login");
+        user.setPassword("password");
+        user.setAge(20);
+    }
+    
+    @Test(expected = RuntimeException.class)
+    public void register_nullUser_runtimeException() {

```

Comment:
Let`s name the tests like this template: methodUnderTest_state_expectedBehavior
Where expectedBehavior is Ok - positive, and NotOk - negative